invenio/testsuite/test_legacy_dbquery.py

```diff
@@ -33,29 +33,31 @@ class TableUpdateTimesTest(InvenioTestCase):
 
     def _check_table_update_time(self, tablename):
         """Helper function to check update time of TABLENAME."""
-        from invenio.base.globals import cfg
-        # detect MySQL version number:
-        res = dbquery.run_sql("SELECT VERSION()")
-        mysql_server_version = res[0][0]
-        if mysql_server_version.startswith("5."):
-            # MySQL-5 provides INFORMATION_SCHEMA:
-            query = """SELECT UPDATE_TIME FROM INFORMATION_SCHEMA.TABLES
-                        WHERE table_name='%s' AND table_schema='%s' """ \
-                        % (tablename, cfg['CFG_DATABASE_NAME'])
-            tablename_update_time = str(dbquery.run_sql(query)[0][0])
-        elif mysql_server_version.startswith("4.1"):
-            # MySQL-4.1 has it on 12th position:
-            query = """SHOW TABLE STATUS LIKE "%s" """ % tablename
-            tablename_update_time = str(dbquery.run_sql(query)[0][12])
-        elif mysql_server_version.startswith("4.0"):
-            # MySQL-4.0 has it on 11th position:
-            query = """SHOW TABLE STATUS LIKE "%s" """ % tablename
-            tablename_update_time = str(dbquery.run_sql(query)[0][11])
-        else:
-            tablename_update_time = "MYSQL SERVER VERSION NOT DETECTED"
-        # compare it with the one detected by the function:
-        self.assertEqual(tablename_update_time,
-                         dbquery.get_table_update_time(tablename))
+        from invenio.ext.sqlalchemy import db
+        if db.engine.driver == 'mysql':
```

Comment:
These are test only for mysql because they are mysql-specific queries.
Anyway, they are tests for legacy dbquery and when finally we'll can remove legacy code and/or definitely pass to postgresql, this testsuite'll not make sense.